### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SunglassJerry**
Updated Sources and Planned Stuff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,19 +64,19 @@ Saikou is crafted based on simplistic yet state-of-the-art elegance. It is an [A
...
 | [MangaSee](https://mangasee123.com)     | SEMI-WORKING |
 | [MangaPill](https://mangapill.com)      | WORKING |
-| [MangaDex](https://mangadex.org)        | WORKING |
+| [MangaDex](https://mangadex.org)        | SEMI-WORKING |
 | [MangaReaderTo](https://mangareader.to) | WORKING |
 | [AllAnime](https://allanime.site)       | WORKING |
 | [Comickfun](https://comick.fun)         | WORKING |
 | [MangaHub](https://mangahub.io)         | WORKING |
+| [MangaKatana](https://mangakatana.com)  | WORKING |
+| [Toonily (18+)](https://toonily.com/)         | WORKING |
```
Comment (scrazzz) [inline README.md:73]:
```suggestion
| [Toonily](https://toonily.com/)         | WORKING |
```
Toonily isn't an NSFW source

**Review by scrazzz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SunglassJerry) [reply README.md:73]: But it does contain a lot of nsfw. A comparatively small portion is non-nsfw. I explored the site for 3 minutes before updating it

Comment (SunglassJerry) [reply README.md:73]: That's why I think it's better if we mark it nsfw. But can be accessed for normal mangas in the app.

Comment (SunglassJerry) [reply README.md:73]: If you still don't want it to be like that. Then ok...

File: README.md [hunk 2]
```diff
@@ -41,15 +41,15 @@ Saikou is crafted based on simplistic yet state-of-the-art elegance. It is an [A
...
 | SITE                                      | STATUS  | DOWNLOADS |
 |:-----------------------------------------:|:-------:|:---------:|
-| [9Anime](https://9anime.pl)               | WORKING | SOME |
+| [9Anime](https://9anime.pl)               | SEMI-WORKING | SOME |
 | [Gogo](https://gogoanime.cm)              | WORKING | SOME |
-| [AllAnime](https://allanime.site)         | WORKING | SOME |
+| [AllAnime](https://allanime.site)         | SEMI-WORKING | SOME |
 | [Zoro](https://zoro.to)                   | WORKING | SOME |
 | [Tenshi](https://tenshi.moe)              | WORKING | YES  |
 | [Kamyroll](https://kamyroll.herokuapp.com)| WORKING | NO   |
```
Comment (scrazzz) [inline README.md:49]:
```suggestion
| [Kamyroll](https://github.com/kamyroll/Kamyroll-Wiki/wiki)| WORKING | NO   |
```

Comment (scrazzz) [inline README.md:49]:
```suggestion
| [Kamyroll](https://github.com/kamyroll/Kamyroll-Wiki/wiki)| WORKING | NO   |
```

**Review by scrazzz — CHANGES_REQUESTED** [review]

**Review by scrazzz — APPROVED** [review]

**Comment by scrazzz** [comment]
LGTM

**Comment by Dawson908** [comment]
@scrazzz app having problem

File: README.md (hunk 2 above)
Comment (Dawson908) [reply README.md]: Need some new sources on the app

**Pull request merged**